src/docs/product/performance/display.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Display Selection"
+sidebar_order: 2
+description: "The Display dropdown sets the layout of the Performance homepage."
+---
+
+The Display dropdown automatically selects a set of [metrics](../metrics/) and table columns based on the platform of the selected project. For example, selecting a JavaScript project will set the Display to Frontend (Pageload). You can also manually change the Display, but note that some modes may not be applicable. For example, the Web Vitals shown in Frontend (Pageload) mode only reflect pageload transactions.
```

Comment:
`You can change the Display, but some modes may not be applicable as they do not apply to the platform.` I like this 👍

Reply:
i changed this to "some modes may not be applicable to the platform" since i'm pretty sure "may not be applicable as they do not apply" is a tautology